### PR TITLE
Use group image in email header

### PR DESCRIPTION
**Pull request by nicksellen**
Closes https://github.com/yunity/karrot-frontend/issues/1770

### CODE REVIEW

**Comment by tiltec** [comment]
Do you think it's ok if the picture breaks sometimes? I imagine the case where the group changes their image and the backend garbage collects the old image. Maybe that's fine, as most users won't look at their old notification emails anyways.

**Comment by nicksellen** [comment]
I guess the fancy way would be to implement a special image fixed URL that gives you the current one,..e.g /groups/1/logo.jpg.

**Comment by nicksellen** [comment]
I just need to add a test to test my changes too...

File: karrot/utils/email_utils.py [hunk 1]
```diff
@@ -96,6 +97,12 @@ def prepare_email(
         raise Exception('email context should not redefine defaults: ' + ', '.join(redefined_keys))
     context.update(default_context)
 
+    if 'header_image' not in context:
+        if 'group' in context and context['group'].photo:
+            context['header_image'] = context['group'].photo.url
+        else:
+            context['header_image'] = settings.DEFAULT_EMAIL_HEADER_IMAGE
```
Comment (tiltec) [inline karrot/utils/email_utils.py]: This seems to result in `{HOSTNAME}https://user-images.githubusercontent.com/31616/36565633-517373a4-1821-11e8-9948-5bf6887c667e.png`, at least I didn't notice anything that should prevent it.

Comment (nicksellen) [reply karrot/utils/email_utils.py]: Ah yeah, of course. Doh!

Comment (nicksellen) [reply karrot/utils/email_utils.py]: Should be fixed now. I'm trying to learn how to program again...

**Pull request merged**